content/en/real_user_monitoring/browser/data_collected.md

```diff
@@ -141,6 +141,7 @@ The following diagram illustrates the RUM event hierarchy:
 |--------------------------------|--------|----------------------------------------------------------------------------------------------------------------|
 | `session.id`                      | string | Randomly generated ID for each session.                                                                      |
 | `session.ip`                      | string | Client ip address. If you want to stop collecting this attribute, [contact Support][8].                                                                       |
+| `session.is_active`                      | string | Indicates if the session is currently active.                                                                       |
```

Comment:
I took this description from https://docs.datadoghq.com/real_user_monitoring/android/data_collected/?tab=session#session-attributes